src/lib/custom.js

```diff
@@ -7,9 +7,18 @@ const { version } = require("../../package.json");
 */
 class Custom {
     constructor(token, baseURL) {
-        /** @access private */
+        /**
+         * The Weeby-API token
```

Comment:
Rename `Weeby-API` to `Weeby API` 😛. Applies to the rest of the files that contain `Weeby-API`